docs/conf.py

```diff
@@ -122,7 +122,6 @@
     'github_user': 'GoogleCloudPlatform',
     'github_repo': 'google-cloud-python',
     'github_banner': True,
-    'travis_button': True,
```

Comment:
1. Why?
1. This is certainly unrelated to Spanner

Reply:
1. Because we do not use Travis, at all, so the button is always pegged to our last-ever Travis build.
  2. Yes, it is. The world will go on. :-)

Reply:
I agree with "The world will go on", but not "Yes, it is". (Good call on "we do not use Travis". MERP.)